backend/kernelCI_app/urls.py

```diff
@@ -24,6 +24,10 @@ def viewCache(view):
          viewCache(views.TreeView),
          name="tree"
          ),
+    path("tree-fast/",
```

Comment:
I'd rather use the same endpoint and pass an optional boolean GET argument from `.env`'s frontend.